transactions/README.md

```diff
@@ -0,0 +1,52 @@
+# Transactions
+
+## Transaction object
+
+```
+{   
+    typeOfCurrency: string,
+    typeOfTransaction: string,
+    userID: string,
+    amount: number,
+    dateTime: unix-timestamp
+}
+```
+
+## **Requests**
+
+|               Route                |           Description           |
+| :--------------------------------: | :-----------------------------: |
+|      [GET /fetchLatest/:username?noOfRecords](#fetch-latestTransactions)      | Returns noOfRecords latest Transactions in the system  for username provided|
+----
+## **GET /latestTransactions**
+
+Returns N number of transactions for username provided. First fetch the userid from users collection and get userId of loggedin user.
+Fetch transactions using userId.
+
+- **Params**  
+  username: username of loggedin user.
+  noOfRecords: No of transactions want to fetch. Default value is 10 if not provided in request URL
+- **Query**  
+  username=[string] noOfRecords=[integer]
```

Comment:
`username` need not be here, since it's a path param
Also, can `noOfRecords` be simply `n`? We can mention what `n` is here itself in brackets

Reply:
Username added in Param and n & o added in Query